README.md

```diff
@@ -136,17 +137,16 @@ If you have installed Chocolatey, you can complete these steps to set up JDK, Ap
 > On OS X systems, you can develop, build, and deploy NativeScript projects that target iOS and Android.
 
 * OS X Mavericks
-* [Node.js 0.10.26] or a later stable official release
+* [Node.js 0.10.26][Node.js 0.10.26] or a later stable official release
 * For iOS development
-	* [iOS 7.0 SDK][12] or a later stable official release
-	* [Xcode 5][12] or a later stable official release
+	* [Xcode 6][12] or a later stable official release
 	* [Xcode command-line tools][12]
```

Comment:
This last comment is unclear. What may we include? the Mono req or that Mono is req-ed for library add or something about library add?

If the comments is about the Mono req being listed as a whole, please, take a look at line 144 - it's already there.

And finally, should it or shoildn't it be 64-bit?